### DISCUSSION TITLE
2.0动态不能自动更新，也没有找到1.0对应设置

### DISCUSSION THREAD

**Discussion by zxr1223** [the1812/Bilibili-Evolved#2855] [提问 / Q&A] [answered]
我现在两台设备，一台用的1.0，一台用的2.0，发现升级2.0之后动态没办法自动更新了，2.0必须要手动刷新页面才能更新动态，1.0就不是这样，但是找了一下也没发现具体对应的设置在哪，请问有没有了解的解答一下？

**Comment by Geeyun-JY3** [comment, marked as answer]
见：#2737
